src/torchrunx/utils.py

```diff
@@ -21,6 +25,23 @@ def get_open_port() -> int:
     return port
 
 
+def automatic() -> tuple[list[str], int]:
+    """
+    Automatically determine allocation sizes
+
+    :return: Allocation hosts and workers per host
+    :rtype: tuple[list[str], int]
+    """
+
+    if "SLURM_JOB_ID" not in os.environ:
```

Comment:
let's actually make a

```python
def in_slurm_job() -> bool:
    return "SLURM_JOB_ID" in os.environ
```

so we can more consistently call that in various places.